src/app/MyPage/Like.tsx

```
@@ -0,0 +1,65 @@
+import { Arrow } from "../../assets";
+import { GlassesLensTab } from "../../components/Shopping";
+import { useState } from "react";
+import ProductCard from "../../components/Shopping/ProductCard";
+import { ScrollView } from "react-native-gesture-handler";
+import { useNavigation } from "@react-navigation/native";
+
+const Like = () => {
+  const navigation = useNavigation();
+
+  const [selectedTab, setSelectedTab] = useState<number>(1)
+  const [counter, setCounter] = useState<number>(1)
```

ESLint

[error] 'setCounter' is assigned a value but never used. Allowed unused elements of array destructuring patterns must match /^_/u.

(@typescript-eslint/no-unused-vars)